test/index.ts

```diff
@@ -6,4 +6,6 @@ export {
   nounsAuctionHouseAbi,
   wagmiMintExampleAbi,
   writingEditionsFactoryAbi,
+  ensAbi,
+  customSolidityErrorsAbi,
```

Comment:
let's export `wethAbi` from here too and make sure any import of it comes from here.